final-project/src/tier1/envs/aerial_robot_finalproject_tier1_task.py

```diff
@@ -106,6 +108,10 @@ def __init__(self, cfg: AerialRobotCfg, sim_params, physics_engine, sim_device,
         self.drone_hit_ground_buf = torch.ones(self.num_envs, device=self.device, dtype=torch.long)
 
         self.depth_image = torch.zeros((1, 1024), device=self.device)
+        self.contact_force_tensor = self.gym.acquire_net_contact_force_tensor(self.sim)
```

Comment:
I am fine leaving this in here but the reason it wasn't was because tier1 wasn't supposed to have objects